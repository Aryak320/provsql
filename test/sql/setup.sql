\set ECHO none
\pset format unaligned
CREATE EXTENSION provsql CASCADE;

DROP EXTENSION provsql;

CREATE EXTENSION provsql;

CREATE SCHEMA provsql_test;
