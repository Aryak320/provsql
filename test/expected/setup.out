\set ECHO none
NOTICE:  installing required extension "uuid-ossp"
